stable/kommander-cert-federation/Chart.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v2
+appVersion: "0.0.2"
+description: A Helm chart to create and federate TLS certificates used by kommander internals
+home: https://github.com/mesosphere/charts
+name: kommander-cert-federation
+type: application
+version: v0.0.2
```

```suggestion
version: 0.0.2
```

`version: A SemVer 2 version (required)` I think we can't use `v` to be semver compatible